README.md

```diff
@@ -15,19 +15,27 @@
 
 ## setup
 
-1. install topiary-cli (0.6.0+ suggested)
-2. clone this repo to `$env.XDG_CONFIG_HOME/topiary`
-3. set environment variables
-
 ```nushell
+# install topiary-cli (0.6.0+ suggested)
+cargo install --git https://github.com/tweag/topiary topiary-cli
```

Comment:
Since cargo is not the only way here (I install it via nix), better to mark this as an example?